integration_tests/mockApis/nonAssociationsApi.ts

```diff
@@ -78,6 +78,20 @@ export default {
     })
   },
 
+  stubGetNonAssociationToUpdate: () => {
+    return stubFor({
+      request: {
+        method: 'GET',
```

Comment:
the update method is `'PATCH'`

Reply:
My thinking here was to 'GET' the two prisoners to _then_ perform the 'PATCH' on. There was already a stub called 'GetNonAssociation' but it was returning the wrong prisoners hence naming this one 'GetNonAssociationToUpdate'.

I have updated those stubs in a recent commit to try and make that clearer ... but I could be doing something unnecessary here, so do let me know if so! 😃

Reply:
aah i misread